src/notification.rs

```diff
@@ -114,11 +115,18 @@ impl Notification {
 
      /// Wrapper for `NotificationHint::ImagePath`
     #[cfg(target_os="windows")]
-    pub fn image_path(&mut self, path:&str) -> &mut Notification {
+    pub fn image_path(&mut self, path: &str) -> &mut Notification {
         self.path_to_image = Some(path.to_string());
         self
     }
 
+    /// app's System.AppUserModel.ID
+    #[cfg(target_os="windows")]
+    pub fn app_id(&mut self, app_id: &str) -> &mut Notification {
```

Comment:
curious, can this be changed for every notification? on mac it can only be done once. Just asking.

Reply:
I think so, but I'm not sure. I can make a quick test if you want me to.

Reply:
Yeah it can be changed for every notification:
![Untitled](https://user-images.githubusercontent.com/20051258/81607251-c2ca9080-93aa-11ea-9929-34d96a9665ba.png)